pydatalab/tests/server/test_samples.py

```diff
@@ -750,3 +750,174 @@ def test_add_items_to_collection_success(client, default_collection, example_ite
     child_refcodes = [item["refcode"] for item in collection_data["child_items"]]
 
     assert all(refcode in child_refcodes for refcode in refcodes)
+
+
+@pytest.mark.dependency()
```

Comment:
```suggestion
```

I would double check, but if these tests really are isolated then you can remove all the dependency markers -- I would just make extra sure they don't have any side effects, i.e., stuff left in the database after the test (which is non-trivial), otherwise you can mark them as having a linear dependency chain by listing the function name above it in this decorator, should at least be deterministic in that case

Reply:
So I've put it back for now. Maybe I'm doing something wrong ?